src/rpc/net.cpp

```diff
@@ -102,6 +102,7 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
             "       ...\n"
             "    ],\n"
             "    \"whitelisted\": true|false, (boolean) Whether the peer is whitelisted\n"
+            "    \"minfee\": n,               (numeric) The minimum fee rate for transactions this peer accepts\n"
```

Comment:
Could be `minfeefilter` to not clash with https://github.com/bitcoin/bitcoin/blob/3c8d1ae15352d5c92d5903536c8fe07f771e97a0/src/rpc/blockchain.cpp#L1720